### PR TITLE
Bs4 card riot

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: docs_src/_includes/layouts/containers/wells.html [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
-  <h2 id="wells" class="page-header">Wells</h2>
-    <p class="bs-callout bs-callout-info">By default singleton wells have a bottom margin of <code>22px</code> to remove this simple add the <code>well-collapse</code> class to your well.</p>
-  <h3 class="status status-chat">Default well</h3>
-  <p>Use the well as a simple effect on an element to give it an inset effect.</p>
-  <div class="well" data-example-id="default-well">
-    <div class="well">
-      Look, I'm in a well!
+
+<h2> Wells </h2>
+<p>Wells are flexible and extensible wrappers for content which separate our gird structures from content.</p>
```
Comment (wjramos) [inline docs_src/_includes/layouts/containers/wells.html:4]: `gird`

File: docs_src/_includes/markup-templates/card/card.tag [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<cedar-card class="{ card: true, card-block: isCardBlock }{ ' ' + mydata}">
```
Comment (megafatsquirrel) [inline docs_src/_includes/markup-templates/card/card.tag:1]: Space is missing from mydata

File: docs_src/assets/tags/docs/cedar-code-toggle.tag [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<cedar-code-toggle
+class="code-toggle"
+data-theme={ theme }
+>
+<div class="code-toggle--controle checkbox {toggle} {align}">
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/docs/cedar-code-toggle.tag]: Spacing

File: docs_src/assets/tags/docs/cedar-example-card.tag [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<cedar-example-card>
+    <cedar-card>
+        <cedar-card-block delimiter="top" theme="light-20">
+            <h2 class="h4">{opts.title}</h2>
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/docs/cedar-example-card.tag]: Spacing

File: docs_src/assets/tags/layouts/cedar-card/cedar-card-block.tag [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<cedar-card-block
+class="card-block {delimiter} {opts.align} {opts.additional}"
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/layouts/cedar-card/cedar-card-block.tag]: Spacing

File: docs_src/assets/tags/layouts/cedar-card/cedar-card.tag [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<cedar-card
+    class="card {spacing} {opts.align} {opts.additional}"
+    data-theme={ opts.theme }
+    >
+    <yield/>
+    <script>
+     if(opts.spacing === "true") {
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/layouts/cedar-card/cedar-card.tag]: Change to single quotes

File: docs_src/assets/tags/layouts/cedar-card/cedar-card-image-cap.tag [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<cedar-card-image-cap
+class="card-media media-frame {ratio} {orientation} {position}">
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/layouts/cedar-card/cedar-card-image-cap.tag]: Spacing

File: docs_src/assets/tags/layouts/cedar-card/cedar-card.tag [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<cedar-card
+    class="card {spacing} {opts.align} {opts.additional}"
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/layouts/cedar-card/cedar-card.tag]: Spacing

File: docs_src/assets/tags/layouts/cedar-card/cedar-card-image-cap-top.tag [hunk 9]
```diff
@@ -0,0 +1,9 @@
+<cedar-card-image-cap-top
+class="card-media-top media-frame {ratio} {orientation} {position}">
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/layouts/cedar-card/cedar-card-image-cap-top.tag]: Spacing

File: docs_src/assets/tags/layouts/cedar-card/cedar-card-image-cap-bottom.tag [hunk 10]
```diff
@@ -0,0 +1,9 @@
+<cedar-card-image-cap-bottom
+class="card-media-bottom media-frame {ratio} {orientation} {position}">
```
Comment (megafatsquirrel) [inline docs_src/assets/tags/layouts/cedar-card/cedar-card-image-cap-bottom.tag]: Spacing

**Pull request closed without merging**